frontend/components/organisms/Navbar/index.tsx

```diff
@@ -57,16 +57,32 @@ const Navbar = (): JSX.Element => {
     }
 
     return (
-        <nav className="z-10 h-[80px] w-full bg-white drop-shadow-md">
-            <div className="flex h-full w-full flex-wrap items-center justify-between">
-                <Link href="/questions" className="flex h-full items-center justify-center">
-                    <div className="ml-[60px] text-[32px] font-bold leading-[38px] tracking-tighter">
-                        <span className="text-primary-red">Sun* </span>
-                        <span className="text-primary-black">Overflow</span>
+        <nav className="h-full w-full bg-white">
+            <div className="flex h-full w-full flex-row items-center justify-between px-8">
+                <Link
+                    href="/questions"
+                    className="group relative flex flex-row items-center justify-center gap-px"
+                >
+                    <div className="z-10 flex h-9 w-9 items-center justify-center">
+                        <Image
+                            height="100"
+                            width="100"
+                            alt="Sun Bear Logo"
+                            src="/images/sun_logo.png"
+                            className="h-8 w-8 transition-all duration-300 group-hover:h-9 group-hover:w-9"
+                        />
                     </div>
+                    <div className="relative z-10 flex flex-col">
+                        <span className="absolute -top-[19px] text-[20px] font-semibold text-primary-red transition-all duration-500 group-hover:text-white">
```

Comment:
```suggestion
                        <span className="absolute -top-[19px] text-xl font-semibold text-primary-red transition-all duration-500 group-hover:text-white">
```

Please apply to all related.
reference: https://tailwindcss.com/docs/font-size

Reply:
Applied